tests/server-tests/api_test.go

```diff
@@ -120,19 +120,6 @@ func testMethod(t *testing.T, testInput TestInput) error {
 		}
 	}
 
-	if testInput.method == "GET" { // Are there other methods which get content. We could genericize
```

Comment:
Are these tests not useful, or do they conflict with the change we're making?

Reply:
Due to a merge issue, that section got added twice, so this leaves the first one and eliminates the repeat.